src/components/PostDetails/PostDetails.scss

```diff
@@ -49,5 +53,6 @@
   &__form-wrapper {
     background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='10' ry='10' stroke='%234D457BFF' stroke-width='4' stroke-dasharray='6%2c 14' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
     border-radius: 10px;
+    margin-top: 50px;
```

Comment:
Don't set margin for top component, it will have unpredictable behaviour. Better wrap this components with `div` and set `gap` for them for example